lib/modules/datasource/sbt-package/index.ts

```diff
@@ -200,6 +193,14 @@ export class SbtPackageDatasource extends MavenDatasource {
       }
     }
 
+    logger.debug(
+      `No versions discovered for ${packageName} listing organization root package folder, fallback to maven datasource for version discovery`
+    );
```

Comment:
Let's log static text and provide relevant data explicitly:

```suggestion
    logger.debug(
      { packageName },
      'Sbt: no versions discovered, falling back to Maven'
    );
```

Reply:
Ah, okay, then please never mind